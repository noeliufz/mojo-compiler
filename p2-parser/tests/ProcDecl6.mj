proc p(var i, j: int) {}
