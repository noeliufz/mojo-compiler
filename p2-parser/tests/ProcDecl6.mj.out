proc p (
    var i: int;
    var j: int;
  )
{
}
